pipeline-model-definition/src/test/resources/json/whenEnvFalse.json

```diff
@@ -31,25 +31,25 @@
           }]
         }]
       }],
-      "when": {"condition": {
+      "when": {"conditions": [      {
         "name": "environment",
-        "arguments": [
+        "arguments":         [
           {
             "key": "name",
-            "value": {
+            "value":             {
               "isLiteral": true,
               "value": "FOO"
             }
           },
           {
             "key": "value",
-            "value": {
+            "value":             {
```

Comment:
I copy-pasted the JSON from our internal `ModelASTElement.toJSON().toString(2)` method, and sometimes that comes out weird in re: spacing.